recipes/moreutils/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set pkgname = "moreutils" %}
+{% set version = "0.59" %}
+{% set sha256 = "a48e11c3197bf79a7bfaa121423e64626e3381d9fedc91d606e9724ae498d1b4" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: http://mirrors.kernel.org/debian/pool/main/m/{{ pkgname }}/{{ pkgname }}_{{ version }}.orig.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+test:
+  commands:
+    - sponge -h
```

Comment:
Would figure `moreutils` would have more? Looks like there is quite a [list]( https://joeyh.name/code/moreutils/ ). Let's add some test commands for them too.